include/fastdds/dds/core/policy/ParameterTypes.hpp

```diff
@@ -1828,6 +1826,35 @@ T get_proxy_property(
 }
 
 } //namespace dds
+
+namespace rtps {
+
+using ParameterId_t = fastdds::dds::ParameterId_t;
```

Comment:
Do we actually need to keep these classes and enum in both namespaces or we can keep only `eprosima::fastdds::dds`?

Reply:
We decided to keep it as the Parameters are also an RTPS concept, it made sense